velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -114,6 +114,18 @@ TEST_P(LeadLagTest, offset) {
       appendColumn(data, makeNullableFlatVector<int64_t>(expectedWindow));
 
   assertQuery(queryInfo.planNode, expected);
+
+  // Out of range offset + default value specified => default value.
```

Comment:
Nit : Write full sentence ending in full-stop. "Out of range offsets return default value, whereas null offsets return null. "